pubnub-gson/src/integrationTest/java/com/pubnub/api/integration/SubscribeIntegrationTests.java

```diff
@@ -0,0 +1,222 @@
+package com.pubnub.api.integration;
+
+import com.pubnub.api.PubNub;
+import com.pubnub.api.callbacks.SubscribeCallback;
+import com.pubnub.api.integration.util.BaseIntegrationTest;
+import com.pubnub.api.models.consumer.PNStatus;
+import com.pubnub.api.models.consumer.objects_api.channel.PNChannelMetadataResult;
+import com.pubnub.api.models.consumer.objects_api.membership.PNMembershipResult;
+import com.pubnub.api.models.consumer.objects_api.uuid.PNUUIDMetadataResult;
+import com.pubnub.api.models.consumer.pubsub.PNMessageResult;
+import com.pubnub.api.models.consumer.pubsub.PNPresenceEventResult;
+import com.pubnub.api.models.consumer.pubsub.PNSignalResult;
+import com.pubnub.api.models.consumer.pubsub.files.PNFileEventResult;
+import com.pubnub.api.models.consumer.pubsub.message_actions.PNMessageActionResult;
+import org.jetbrains.annotations.NotNull;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import static com.pubnub.api.integration.util.Utils.randomChannel;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class SubscribeIntegrationTests extends BaseIntegrationTest {
+
+    private PubNub mGuestClient;
+
+    @Override
+    protected void onBefore() {
+        mGuestClient = getPubNub();
+    }
+
+    @Override
+    protected void onAfter() {
+
+    }
+
+    @Test
+    public void testSubscribeToMultipleChannels() {
+        final String channel1 = randomChannel();
+        final String channel2 = randomChannel();
+        final String channel3 = randomChannel();
+
+        pubNub.subscribe()
+                .channels(Arrays.asList(channel1, channel2, channel3))
+                .withPresence()
+                .execute();
+
+        pause(2);
+
+        assertEquals(3, pubNub.subscribedChannels.size());
+        assertTrue(pubNub.subscribedChannels.contains(channel1));
```

Comment:
It doesn't work for me, I am getting:
`Cannot resolve symbol 'subscribedChannels'`
I have this error in many places in this file as well as in GroupManagementIntegrationTests